pkgs/development/libraries/zlog/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, pkgs }:
+
+stdenv.mkDerivation rec {
+  version = "1.2.12";
+  name = "zlog-${version}";
+
+  src = pkgs.fetchurl {
+    url = "https://github.com/HardySimpson/zlog/archive/${version}.tar.gz";
+    sha256 = "085c5gjz6dbvj7mffz6h2wv1s3pk26xbwfam4mq6q4sdyyii8q4w";
+  };
+
+  configurePhase = ''
+    sed -i -r 's;-Werror;;' src/makefile
+  '';
+
+  buildPhase = ''
+    mkdir -p $out
+    make PREFIX=$out
+  '';
+
+  installPhase = ''
+    make PREFIX=$out install
+  '';
+
+  meta = with stdenv.lib; {
+    description= "reliable, high-performance, thread safe, flexible, clear-model, pure C logging library.";
```

Comment:
First point in https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#making-patches.